src/bubo.h

```diff
@@ -23,10 +24,10 @@ class Bubo : public Nan::ObjectWrap {
     JS_METHOD_DECL(Add);
     JS_METHOD_DECL(Contains);
     JS_METHOD_DECL(Delete);
-    JS_METHOD_DECL(DeleteBucket);
-    JS_METHOD_DECL(GetBuckets);
     JS_METHOD_DECL(Stats);
     JS_METHOD_DECL(Test);
 
-	BuboCache cache_;
+    AttributesTable* at;
```

Comment:
Conventions would indicate this should be called `attrs_table_` -- the trailing underscore indicates it's an instance variable.